src/json_reporter.cc

```diff
@@ -49,6 +49,12 @@ std::string FormatKV(std::string const& key, int64_t value) {
   return ss.str();
 }
 
+std::string FormatKV(std::string const& key, int value) {
```

Comment:
if this is for outputting the args (which i don't think we do other than as part of the name) then it should be int64_t value, right?

Reply:
This fixed a build error so that either type would work.

Reply:
J/K, that error went away once I made your other suggested changes.